File: app/src/main/java/com/drivingschoolrwandaapp/models/request/PasswordChangeRequest.java
package com.drivingschoolrwandaapp.models.request;

import com.google.gson.annotations.SerializedName;

public class PasswordChangeRequest {

    @SerializedName("currentPassword")
    private String currentPassword;

    @SerializedName("newPassword")
    private String newPassword;

    @SerializedName("confirmPassword")
    private String confirmPassword;

    public PasswordChangeRequest(String currentPassword, String newPassword, String confirmPassword) {
        this.currentPassword = currentPassword;
        this.newPassword = newPassword;
        this.confirmPassword = confirmPassword;
    }

    public String getCurrentPassword() {
        return currentPassword;
    }

    public String getNewPassword() {
        return newPassword;
    }

    public String getConfirmPassword() {
        return confirmPassword;
    }
}

File: app/src/main/java/com/drivingschoolrwandaapp/models/request/PasswordChangeRequest.kt
package com.drivingschoolrwandaapp.models.request

import com.google.gson.annotations.SerializedName

data class PasswordChangeRequest(
    @SerializedName("currentPassword") var currentPassword: String = "",
    @SerializedName("newPassword") var newPassword: String = "",
    @SerializedName("confirmPassword") var confirmPassword: String = ""
)
